test/test_heuristic.cpp

```diff
@@ -159,7 +159,7 @@ TEST_P(HeuristicTest16Q, Disjoint) {
   Configuration settings{};
   settings.layering = Layering::DisjointQubits;
   ibmQX5Mapper->map(settings);
-  ibmQX5Mapper->dumpResult(GetParam() + "_heuristic_qx5_dynamic.qasm");
+  ibmQX5Mapper->dumpResult(GetParam() + "_heuristic_qx5_disjoint.qasm");
```

Comment:
Good catch!

Would it make sense to also add a corresponding test for the heuristic mapper using the new layout strategy here?
Maybe this also helps to get the Coverage run in the green.

Reply:
I originally planned on doing so. (That is also why I catched that wrong file name)
However, using Disjoint2qBlocks layering that test would result in an extremely long running mapping for the circuit `qft_16` (`Heuristic/HeuristicTest16Q.Disjoint/qft_16`), because of a pre-existing bug with the ordering of A* search nodes. Since that bug is otherwise completely separate from the layering implementation, I just left that test out.

I am currently working on a solution for that bug. So maybe it makes sense to hold out on merging this PR (including the test) until after.

Reply:
Yeah. I'll wait with the merge here until that bug is fixed. The changes here all look good. So once the bug is fixed and the tests are added, this should be good to go.

Reply:
The tests are now added. Should be ready to be merged